### PR TITLE
PMM-9999 upgrade victoriametrics to 1.77.1 (FB)

### PR DESCRIPTION
**Pull request by idoqo**
https://jira.percona.com/browse/PMM-9999

This is the feature build that contains changes from pmm-server. A separate PR to update the VM version used in the pmm-submodules repo is at https://github.com/Percona-Lab/pmm-submodules/pull/2520

- [ ] https://github.com/percona/pmm-server/pull/360

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2518-65548d2
client docker - perconalab/pmm-client-fb:PR-2518-65548d2
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2518-65548d2.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2518-65548d2&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2518-65548d2.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/9989/

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/15564/

File: ci.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+deps:
```
Comment (BupycHuk) [inline ci.yml:1]: please create one more PR without this file and make that ready for review.

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2518-00bb957
client docker - perconalab/pmm-client-fb:PR-2518-00bb957
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2518-00bb957.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2518-00bb957&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2518-00bb957.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/10030/

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/15687/

**Pull request closed without merging**